package.json

```diff
@@ -23,24 +23,8 @@
     },
     "workspaces": [
         "calendly",
```

Comment:
did this clear all the other workspaces?

Reply:
Yea, to test only the folders I copied into `fake-app-integrations` root folder